alacritty_terminal/src/tty/mod.rs

```diff
@@ -18,6 +16,30 @@ pub mod windows;
 #[cfg(windows)]
 pub use self::windows::*;
 
+/// Configuration for the `Pty` interface.
+#[derive(Clone, Debug, PartialEq, Eq, Default)]
+pub struct PtyConfig {
+    /// Shell options..
+    ///
+    /// [`None`] will use the default shell.
+    pub shell: Option<Shell>,
+
+    /// Shell startup directory.
+    pub working_directory: Option<PathBuf>,
+
+    /// Remain open after child process exits.
+    pub hold: bool,
```

Comment:
This is an `alacritty_terminal` option to drain the PTY? Or are there other reasons?

Reply:
It's a cli flag we pass to hold, it's not a part of any configs.